java/src/main/java/com/google/oak/crypto/SenderEncryptor.java

```diff
@@ -0,0 +1,83 @@
+//
```

Comment:
I'd suggest adding a test class for this and expanding on it as more code is added here.

Reply:
Will be done in a separate PR as part of https://github.com/project-oak/oak/issues/3644

Because we first need the C++ encryption to use it in Java via JNI

Reply:
Is it needed even for a unit test? Having a small unit test will ensure that you won't forget the tests in the future as you update the code. But up to you :)

Reply:
ah, you meant to create an empty test?
will add